### ISSUE TITLE
Unable to select encoder type after update to org.freedesktop.Platform.GStreamer.gstreamer-vaapi

### ISSUE THREAD

**Issue by euclip** [flathub/com.obsproject.Studio.Plugin.Gstreamer#10]
On the 22nd I received this batch of flatpak updates:

```
joulu 22 10:12:26  deploy update  org.freedesktop.Platform.GL.default                 21.08       system       flathub
joulu 22 10:12:26  deploy update  org.freedesktop.Platform.GStreamer.gstreamer-vaapi  21.08       system       flathub
joulu 22 10:12:26  deploy update  org.kde.Platform.Locale                             5.15-22.08  system       flathub
joulu 22 10:12:28  deploy update  org.kde.Platform                                    5.15-22.08  system       flathub
```

After these updates, the **encoder type** field in OBS' output settings (when using Gstreamer Encoder H.264) is empty, and doesn't present anything for me to select when clicked on. This means I can't use hardware accelerated encoding, which was working fine before these updates.

I've seen the same problem before, and then the solution was to install the `org.freedesktop.Platform.GStreamer.gstreamer-vaapi` flatpak, but obviously it's already installed this time.

Here is a screenshot of what I see in OBS: ![output-settings](https://user-images.githubusercontent.com/45699492/209479512-042ad6b6-5048-4c7e-80c5-43b2aa02b0c1.jpg)

I am using Fedora 37 Workstation, and I have a Radeon RX 6800 XT graphics card.

**Comment by marvin0815** [comment]
Not a plugin nor an OBS Flatpak issue but upstream. It looks like a broken Mesa Gallium driver. Possible fix here: https://github.com/obsproject/obs-studio/issues/7956#issuecomment-1364704292

**Comment by PolyCatGamedev** [comment]
Same problem here, and the fix didn't work. I had a conversation with some devs on the obs discord (Tuna and Cookie) and we arrived at the conclusion that the problem comes from the plugin itself not using the correct `org.kde.Sdk` version. Cookie looked at the repo and noticed that the plugin still uses `org.kde.Sdk//6.3` when it should be using `org.kde.Sdk//6.4` or newer.

**Comment by PolyCatGamedev** [comment]
btw. Does anyone know how to reverse the `sudo flatpak update --commit=927c51d661eb70910a3a51e3e05e923cbaadd2c8be77f0bd7fedb18b9c079ce7 runtime/org.freedesktop.Platform.GL.default/x86_64/21.08` command. It broke the plugin alltogether.

**Comment by PolyCatGamedev** [comment]
I'm here with an update on my end. The extension updated, but the still not working. After a bit of looking around, I saw that for the plugin to work, I have to install `runtime/org.freedesktop.Platform.GStreamer.gstreamer-vaapi/x86_64/22.08` front and center on the plugins main page. After installing, it started working again.

So this message is here for people like me that missed the really obvious instructions and to say to any maintainers that this case has been solved on my end.
